### ISSUE TITLE
In standalone mode the link to the pilet is pointing to an relative and not an abolut path 

### ISSUE THREAD

**Issue by Condisil** [smapiot/piral#503]
# Bug Report

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

- [ ] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x ] Are you running the latest version?
- [ x] Did you perform a search in the issues?

## Environment Details and Version

Piral 0.14.19
Windows 10
Chrome

## Description
Pilet location is linked relative:
https://github.com/smapiot/piral/blob/cc5c00f21b8ed4e09a8bb65b8aed04f0deeffb9c/src/tooling/piral-cli/src/apps/build-pilet.ts#L41

## Steps to Reproduce

1) Create an pilet in standalone mode.
2) Serve the pile (pliet will be loaded by http://domain/test-pilet/index.js)
3) navigate to an sub route e.f http://domain/test/subpath
4) reload  the page
5)  pilet could not be loaded error message: "Could not find the requested page. Are you sure it exists?" (pliet will be loaded by  http://domain/test/test-pilet/index.js)

## Expected behavior

Pilet should be loaded from domain root path.

## Actual behavior

Pilet will be loaded from a relative path.

## Possible Origin / Solution

Remove the leading dot  from the link path so that the pilet will be loaded by the root of the domain.

**Comment by FlorianRappl** [comment]
Yeah sounds great to me - would be awesome if you could have a look at it!

I imagine just `/` works but for consistency we should make that "dynamic" such that `createMetadata` takes a `publicPath` variable, which (right now) is set to `/` from `pilet build`.
